_maps/map_files/RandomZLevels/fileList.txt

```diff
@@ -8,16 +8,18 @@
 #DO tick the associated code file for the away mission you are enabling. Otherwise, the map will be trying to reference objects which do not exist, which will cause runtime errors!
 
 #_maps/map_files/RandomZLevels/academy.dmm
-_maps/map_files/RandomZLevels/beach.dmm
+#_maps/map_files/RandomZLevels/beach.dmm
 #_maps/map_files/RandomZLevels/blackmarketpackers.dmm
 #_maps/map_files/RandomZLevels/challenge.dmm
 #_maps/map_files/RandomZLevels/centcomAway.dmm
 #_maps/map_files/RandomZLevels/clownplanet.dmm
 #_maps/map_files/RandomZLevels/example.dmm
-_maps/map_files/RandomZLevels/listeningpost.dmm
-_maps/map_files/RandomZLevels/moonoutpost19.dmm
+#_maps/map_files/RandomZLevels/listeningpost.dmm
```

Comment:
Um.. don't do this, please, or, at least, move all of the usable away missions away from the already-disabled away missions.